repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/contact-item.ftl

```diff
@@ -4,7 +4,7 @@
     <div class="nhsuk-contact">
         <div class="nhsuk-contact__content">
             <#if singleContact && block.title?has_content>
-                <h2>${block.title}</h2>
+                <h2 data-anchorlinksignore="true">${block.title}</h2>
```

Comment:
I guess we may not need `data-anchorlinksignore="true"` for `<h2>` as Will still seem to want this to be included in anchor links (similar to Related Content heading):

> - we’re also going to need exceptions for the contacts component. Contact name h3s are displaying in the list:
> - But let’s keep the “Contacts” h2, as that is helpful to have - as is the Related content header (“Related information” in the screenshot example above)

Reply:
Thanks @smanivasag - I've removed this.